packages/styles/schematics/utils/css/css-classes-update.ts

```diff
@@ -7,6 +7,18 @@ export abstract class CssClassesUpdate {
     abstract searchValue: string;
     abstract replaceValue: string | Replacer;
 
+    get tag(): RegExp | null {
+        return this.tagSelector ? new RegExp(`^${this.tagSelector}$`) : null;
+    }
+
+    get class(): RegExp | null {
+        return this.classSelector ? new RegExp(`^${this.classSelector}$`) : null;
```

Comment:
In the majority of cases the class selector matches the search value, therefore it might be good for this getter to return `this.searcher` instead of `null` if no class selector is defined. What do you think?

Reply:
The `class` property only act as a filter for the existing updates and we're not aware of what is defined in the `searcher` at this point.
So if the `classSelector` is not defined, the `class` filter should be `null`, to not filtering the nodes by something else (like the  `searcher` property), and therefore avoid side-effects.

Reply:
The searcher aims to find a class that needs to be updated, in most cases if an item does not contain that class it can be filtered out.

Looking at the form migrations, the `classSelector` almost always matches the `searchValue` (custom select menu, custom checkbox, custom radio, custom control input, custom control label, custom inline control, switch , toggle switch)

In special cases such as `.form-text`, the classSelector can always be overridden

Reply:
But if you think setting a classSelector for each migration is clearer, that's fine with me too

Reply:
I will as it is for the moment (never touch a running system).